package.yaml

```diff
@@ -30,12 +30,16 @@ library:
   - generics-eot
   - haskeline
   - http-client
+  - ipld-cid
   - megaparsec
   - mtl
+  - lens
```

Comment:
Are we using Prisms? In the spirit of smaller build times, might be worth using micro-lens here, if we aren't using Prisms etc.

Reply:
Though I just see you're using `wreq`, which I think depends on `lens` anyway, so maybe ignore..

Reply:
I’m switching to `microlens` in our code. Unfortunately `wreq` still has the `lens` dependency. We will get of `wreq` soon.